### PR TITLE
Expose inferred and "indicator" historical Rt values through the API timeseries

### PR DESCRIPTION
**Pull request by davidstrauss**
This PR addresses issue #309

### Please Check
- [ ] Are the [JSON schemas](https://github.com/covid-projections/covid-data-model/tree/master/api/schemas) updated?
- [ ] Are the [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md) updated?
- [ ] Are tests passing?

### CODE REVIEW

**Comment by davidstrauss** [comment]
Happy to remove the whitespace changes if it helps. My editor/IDE always attempts to ensure the last line of every file has a newline.

File: api/can_api_definition.py [hunk 1]
```diff
@@ -122,6 +122,12 @@ class CANPredictionTimeseriesRow(pydantic.BaseModel):
         ...,
         description="Total ventilator capacity."
     )
+    Rt: float = pydantic.Field(
+        ..., description="Historical or Inferred Rt"
+    )
+    RtCI90: float = pydantic.Field(
+        ..., description="Rt standard deviation"
```
Comment (ghop02) [inline api/can_api_definition.py:129]: should this be "R_t 90th percent confidence interval"?

Comment (davidstrauss) [reply api/can_api_definition.py:129]: It's copied from the description for the field earlier in the file (approx line 30-35):
```
    RtCI90: float = pydantic.Field(
        ..., description="Rt standard deviation"
    )
```

Easy to switch to however we'd like to describe it, though.

**Comment by davidstrauss** [comment]
Follow-up for API changes in #312

**Pull request merged**